api/handler/object_list.go

```diff
@@ -160,21 +161,28 @@ func fillPrefixes(src []string, encode string) []CommonPrefix {
 	return dst
 }
 
-func fillContents(src []*layer.ObjectInfo, encode string) []Object {
+func fillContentsWithOwner(src []*layer.ObjectInfo, encode string) []Object {
```

Comment:
Does this function make sense? I assume we can use  only fillContents with fetchOwner bool (as you implemented below), but in case of ListObjectsV1 pass a value "false".

Reply:
If we pass `true/false` explicitly, we will need to look at the implementation of the function to understand what this parameter means.

Reply:
But anyway we will need to look at the implementation and `fillContents` is not too hard to understand, I'm not sure it's worth it to create one more wrapper function. But it's just a matter of taste, so that's at your discretion.